app/code/core/Mage/Core/etc/config.xml

```diff
@@ -326,6 +326,9 @@
             <csrf>
                 <use_form_key>1</use_form_key>
             </csrf>
+            <cache>
+                <flush_cron_expr>30 3 * * 6</flush_cron_expr>
```

Comment:
Ok, I will revert it to original value. But just for my information, is there any reason to flush all cache daily?
In our experience once a week is enough (and it is done just for sure) and there was not such cron before 1.7.x at all.

Reply:
I don't see any reason to flush all cache daily. It can be done manually from Backend when needed and one time per week should be fine.

Please note the original value was (30 3 * * 6). As I see the new change is (30 2 * * *) which runs the cron job every day of the week at 2:30 am.

Reply:
All right then. His explanation in Backend is more than enough. The PR looks good to me.

Reply:
Yes, original value is 30 2 * * *  since magento 1.7.x (i do not know the reason, maybe it can "solve" potentital problems with cache records that never expires).
There was not such cron before 1.7.x, we have no problems without that feature that on 1.6.x for many years.

I understand this remark that some people can be relying on it currently and that is the reason the default value should be left as it was. I am ok with that, we have changed the value in our production environment.

But a lot (or some people) probably do not know about that feature at all so they can register some problems under heavy load at a certain redis configuration as we were.
![image](https://user-images.githubusercontent.com/53212098/168569947-120fada6-c608-4181-ada2-47af82fec578.png)

Reply:
I see, our website traffic is very low at 2am so we never face issues.

Since the PR requires at min 2 maintainers approvals, I have no objection if someone else thinks its better to change it to weekly :)